bubblewrap.c

```diff
@@ -3174,6 +3213,14 @@ main (int    argc,
                          -1, FALSE, FALSE);
```

Comment:
Alex's version had this `write_uid_gid_map()` call conditional on `ns_uid != opt_sandbox_uid || ns_gid != opt_sandbox_gid`, but I'm not 100% sure why, and I found that keeping the conditional broke my sandboxed process's ability to connect to D-Bus - I think because my uid wasn't mapped into the container correctly. It's less diff like this, anyway.

Reply:
I think I did that because previously in that case we would not hit this codepath so I would avoid doing what we didn't do before. But we just set up a new user namespace, so we *have* to do this here. So, that change is correct.